app/models/cloud_tenant.rb

```diff
@@ -30,6 +30,8 @@ class CloudTenant < ApplicationRecord
   has_many   :flavors, :through => :cloud_tenant_flavors
   has_many   :cloud_volume_types, :through => :ext_management_system
 
+  validates :name, :length => {:maximum => 128}
```

Comment:
I think I mentioned this elsewhere, but any data that can come from providers I don't think we should be as conservative with, otherwise we run the risk of failing during provider collection.  On the backend, I think we can be much looser with these values.  For example, here's I'd be ok with 1k or even higher.  The frontend is only for new objects, and those forms (I think) are provider specific, so the value could be different on different forms depending on the provider. I understand the need to limit it to _some_ value, but I'd prefer if it were a really large, but reasonable value.

@agrare Thoughts?